src/templates/Resources/DocumentReference.hbs

```diff
@@ -6,7 +6,7 @@
         "type":
         	{{#with (elementAt (toArray docref.ClinicalDocument.code) 0)}}
         		{{>DataType/CodeableConcept.hbs code=this}},
-           	{{/with}},
+           	{{/with}}
```

Comment:
My bad. We should have kept this comma, otherwise it might cause issue in case param for with is empty.